### ISSUE TITLE
Make it clear that other versions of Java are not acceptable

### ISSUE THREAD

**Issue by mushishi78** [microclimate-devops/xml-language-server#19]
To use in VS Code, the `xmlLang.javaHome` has to be set to a version where the sdk explicitly has 1.8 in it:

https://github.com/microclimate-devops/xml-language-server/blob/045c25635e0c1c6919359b43d813edfddbe0f0a7/client-vscode/xml-client/client/extension.js#L108

This took me a while to work out because not being familiar with the Java world, I assumed that my "java-11-openjdk-amd64" install would do. I worked out how to open the developer tools in VS Code, found the console output from the line above and twigged what was going on. I then installed a "java-8-oracle" and set my `xmlLang.javaHome` to that and it's seems to be working.

Perhaps an error message saying that you found a version of java, but the version output did contain 1.8 so it's not to be trusted? Or some documentation saying that versions that appear to be later than that are not accepted?

Thanks anyway, look forward to using it :D

**Comment by jsamr** [comment]
Yeah I think this is an error and should be replaced with a `>=` number comparison ; isn't it backward compatible ?
